src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Deprecate using `DateTime` or `DateTimeImmutable` model data with a different timezone than configured with the
    `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
+ * Deprecate `FormEvent::setData()` for events that do not allow it
```

Comment:
I think we can list both events here. And we then also need to add this to the upgrade file.

Reply:
Done, thanks!